connectors/src/lib/models/slack.ts

```diff
@@ -21,6 +21,7 @@ export class SlackConfigurationModel extends Model<
   declare botEnabled: boolean;
   declare connectorId: ForeignKey<ConnectorModel["id"]>;
   declare whitelistedDomains?: readonly string[];
+  declare whiteListedChannelPattern?: string;
```

Comment:
Given that you have `allowNull: true` the type should reflect this.
```suggestion
  declare whiteListedChannelPattern?: string | null;
```